relay-server/src/endpoints/project_configs.rs

```diff
@@ -129,15 +131,19 @@ async fn inner(
 
     let mut configs = HashMap::with_capacity(keys_len);
     let mut pending = Vec::with_capacity(keys_len);
-    let global_config = match inner.global {
-        true => Some(
-            state
-                .global_config()
-                .send(global_config::Get)
-                .await?
-                .get_ready_or_default(),
-        ),
-        false => None,
+    let (global, global_status) = match inner.global {
+        true => match state.global_config().send(global_config::Get).await? {
```

Comment:
Maybe make this an `if`, I think we talked about this and we prefer `if` over match on boolean